src/components/Animated/SlideEntry.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import cx from 'classnames'
+import styles from './SlideEntry.module.scss'
+
+class SlideEntry extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      entered: false
+    }
+
+    setTimeout(() => {
```

Comment:
1. An uncleared timer will cause to the dangling pointer and change the state of the unmounted component.
2. No need to create this component. There is a great [library](https://github.com/reactjs/react-transition-group) (reliable and well documented) for custom animations. Their examples contain scenarios similar to ours

Reply:
About 2. @DmitriVanGuard We have animate-component library inside our app. Or better switch to https://popmotion.io/pose/ (Did you see it).
But for small interaction, who knows what is better.

Reply:
About timer yes, it's a warning.

Reply:
Speaking in the context of moving to our custom animations:
1. [react-transition-group](https://bundlephobia.com/result?p=react-transition-group@2.7.0)(4.8 kB) is ~5x smaller gzipped than [react-pose](https://bundlephobia.com/result?p=react-pose@4.0.8)(20.8 kB);
2. `react-transition-group` is recommended in the React [documentation](https://reactjs.org/docs/animation.html);
3. `react-pose` really fits for `styled-components` meanwhile we are using css modules;

Reply:
Ok. Let's use react-transition-group